client/my-sites/category-selector/index.jsx

```diff
@@ -136,13 +136,26 @@ module.exports = React.createClass( {
 			disabled = true;
 		}
 
-		input = (
-			<input id={ domId } type={ inputType } name='categories'
-				value={ itemId }
-				onChange={ this.props.onChange.bind( null, item ) }
-				disabled={ disabled }
-				checked={ checked } />
-		);
+		if ( !this.props.multple ) {
```

Comment:
@mtias I'm not sure why I had to add `!` for it to show checkboxes instead of radio buttons when I made this change. I'd like to get it right before proceeding to fix the other instances I linked in the above comment on this commit.